### DISCUSSION TITLE
Native support for other database-related objects is expected?

### DISCUSSION THREAD

**Discussion by apabolleta-dasnano** [sqlalchemy/sqlalchemy#11730] [Ideas]
Native support for other database-related objects is expected, such as views, triggers or stored procedures? Even if these objects vary from one database backend to another, I think sqlalchemy core/ORM API would help to write most use cases using database agnostic SQL syntax as DML or DDL statements with little change. As known for the moment, currently supported approaches for the previous object creation and management is to write database-specific syntax as textual SQL expressions.

**Comment by CaselIT** [comment]
Hi,

For views the answer is maybe, but there isn't any planned activity at the moment.
For the other mentioned features is more likely a no.

I think most of what you seek is in the sqlalchemy-utils package, but IIRC the support may be postgresql first.

@zzzeek not sure if you have further considerations to add on this
